doc/source/whatsnew/v0.19.0.txt

```diff
@@ -473,6 +473,130 @@ New Behavior:
 
    type(s.tolist()[0])
 
+.. _whatsnew_0190.api.series_ops:
+
+``Series`` operators for different indexes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Following ``Series`` operators has been changed to make all operators consistent,
+including ``DataFrame`` (:issue:`1134`, :issue:`4581`, :issue:`13538`)
+
+- ``Series`` comparison operators now raise ``ValueError`` when ``index`` are different.
+- ``Series`` logical operators align both ``index``.
+
+.. warning::
+   Until 0.18.1, comparing ``Series`` with the same length has been succeeded even if
+   these ``index`` are different (the result ignores ``index``).
+   As of 0.19.1, it raises ``ValueError`` to be more strict.
+
```

Comment:
As I said in another comment, I would be more explicit here about how to deal with this when you used such code (so in case you wanted this behaviour, you now have to use `s1.values == s2.values`)

Maybe an example where you show all 4 cases (old behaviour, new default, new .values, new flex method) next to each other. The overview below is very structured, which is certainly good to keep! But I would also give one example for probably the case where most likely breakages will occur that compares all different behaviour together.